fe/fe-core/src/main/java/org/apache/doris/analysis/ShowDataStmt.java

```diff
@@ -270,6 +371,15 @@ public String toSql() {
             return builder.toString();
```

Comment:
Maybe it should not be returned here.
The reason is that even if there is no tableName, there may be an order by clause.